src/app/submission/components/metadata-details-dialog/metadata-details-dialog.component.ts

```diff
@@ -45,58 +37,53 @@ export class MetadataDetailsDialogComponent implements OnInit {
       'schema_type',
       'provenance'
     ],
+    viewMode: false,
     showCancelButton: false,
     showResetButton: false,
     submitButtonLabel: 'Save'
   };
-  saving: boolean = false;
 
-  constructor(private ingestService: IngestService,
-              private alertService: AlertService,
+  constructor(private alertService: AlertService,
               public dialogRef: MatDialogRef<MetadataDetailsDialogComponent>,
               @Inject(MAT_DIALOG_DATA) public dialogData: any) {
   }
 
   ngOnInit(): void {
+    if (!this.dialogData.hasOwnProperty('schema')) {
+      this.alertService.error('Error: No schema to start form!', 'This is probably our fault rather than yours, please contact your wrangler or developer.');
```

Comment:
```suggestion
      this.alertService.error('Error: No schema to start from!', 'This is probably our fault rather than yours, please contact your wrangler or developer.');
```